prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/bundle1.txt

```diff
@@ -0,0 +1 @@
6bfd9e9d77"}}},{"fullUrl":"ServiceRequest/9f0d01d6-c1c6-4459-8373-f51f0e39b2b7","resource":{"resourceType":"ServiceRequest","id":"9f0d01d6-c1c6-4459-8373-f51f0e39b2b7","status":"completed","intent":"order","code":{"coding":[{"system":"http://loinc.org","code":"97099-6"}]},"subject":{"reference":"Patient/3aaba67c-bd90-4b41-a820-816bfd9e9d77"},"requester":{"reference":"PractitionerRole/15c2063b-ada9-48e2-b7a5-ef41e1d04f2f"},"performer":[{"reference":"Organization/b74499a0-2abe-4ea1-8cb5-496906c11852"}]}},{"fullUrl":"Device/78dc4d98-2958-43a3-a445-76ceef8c0698","resource":{"resourceType":"Device","id":"78dc4d98-2958-43a3-a445-76ceef8c0698","manufacturer":"Becton, Dickinson and Company (BD)","deviceName":[{"name":"1copy COVID-19 qPCR Multi Kit","type":"model-name"}]}},{"fullUrl":"PractitionerRole/15c2063b-ada9-48e2-b7a5-ef41e1d04f2f","resource":{"resourceType":"PractitionerRole","id":"15c2063b-ada9-48e2-b7a5-ef41e1d04f2f","practitioner":{"reference":"Practitioner/c712873f-0752-4047-a8b3-dc276c009e69"},"organization":{"reference":"Organization/b74499a0-2abe-4ea1-8cb5-496906c11852"}}},{"fullUrl":"Observation/4301a340-384e-4760-938c-cee7418c26f4","resource":{"resourceType":"Observation","id":"4301a340-384e-4760-938c-cee7418c26f4","status":"final","code":{"coding":[{"system":"http://loinc.org","code":"94500-6"}],"text":"COVID-19"},"subject":{"reference":"Patient/3aaba67c-bd90-4b41-a820-816bfd9e9d77"},"performer":[{"reference":"Organization/b74499a0-2abe-4ea1-8cb5-496906c11852"}],"valueCodeableConcept":{"coding":[{"system":"http://snomed.info/sct","code":"260373001"}]},"specimen":{"reference":"Specimen/52a582e4-d389-42d0-b738-bee51cf5244d"},"device":{"reference":"Device/78dc4d98-2958-43a3-a445-76ceef8c0698"}}},{"fullUrl":"Observation/96eb39a3-11be-461f-931c-0679d5ee510d","resource":{"resourceType":"Observation","id":"96eb39a3-11be-461f-931c-0679d5ee510d","status":"final","code":{"coding":[{"system":"http://loinc.org","code":"LP14239-5"}],"text":"Flu A"},"subject":{"reference":"Patient/3aaba67c-bd90-4b41-a820-816bfd9e9d77"},"performer":[{"reference":"Organization/b74499a0-2abe-4ea1-8cb5-496906c11852"}],"valueCodeableConcept":{"coding":[{"system":"http://snomed.info/sct","code":"260415000"}]},"specimen":{"reference":"Specimen/52a582e4-d389-42d0-b738-bee51cf5244d"},"device":{"reference":"Device/78dc4d98-2958-43a3-a445-76ceef8c0698"}}},{"fullUrl":"Observation/657d9c1d-13d4-49d5-97fb-3fd629ed88cc","resource":{"resourceType":"Observation","id":"657d9c1d-13d4-49d5-97fb-3fd629ed88cc","status":"final","code":{"coding":[{"system":"http://loinc.org","code":"LP14240-3"}],"text":"Flu B"},"subject":{"reference":"Patient/3aaba67c-bd90-4b41-a820-816bfd9e9d77"},"performer":[{"reference":"Organization/b74499a0-2abe-4ea1-8cb5-496906c11852"}],"valueCodeableConcept":{"coding":[{"system":"http://snomed.info/sct","code":"260373001"}]},"specimen":{"reference":"Specimen/52a582e4-d389-42d0-b738-bee51cf5244d"},"device":{"reference":"Device/78dc4d98-2958-43a3-a445-76ceef8c0698"}}}]}
```

Comment:
I cannot find in the code where this file is being used. Delete? If you are using this file for unit tests then this is not the right place for it since this is the integration test folder.

Reply:
Removed